app/controllers/drivers_controller.rb

```diff
@@ -5,7 +5,9 @@ class DriversController < ApplicationController
 
   def orders
     @driver = current_driver
-    @orders = @driver.orders.all
+    @orders = @driver.orders.where(status: 'waiting')
+    @orders = @driver.orders.where('updated_at >=? AND status !=?',
+                                   Date.yesterday, 'waiting') if @orders == []
```

Comment:
It's more semantic to use `.blank?` instead of ` == []` here :construction_worker_man: